autogpt_platform/frontend/src/components/integrations/credentials-input.tsx

```diff
@@ -91,14 +94,10 @@ export const CredentialsInput: FC<{
     useState<AbortController | null>(null);
   const [oAuthError, setOAuthError] = useState<string | null>(null);
 
-  if (!credentials) {
+  if (!credentials || credentials.isLoading) {
     return null;
   }
 
-  if (credentials.isLoading) {
-    return <div>Loading...</div>;
-  }
-
```

Comment:
why this change?

Reply:
So it doesn't show "Loading..." and then disappear if there's one credentials (it's chosen by default and nothing is displayed)